### PR TITLE
Update positions 0725

### PR DESCRIPTION
**Pull request by Soline324**
* Update job descriptions of engineering-efficiency;
* Add a new tag named "Product & Design";
* Add three new positions to"Product & Design", including database-product-manager, website-product-manager and graphic-designer;
* Delete tidb product manager;
* Please help to add a new avatar for designer.

@YiniXu9506 PTAL

### CODE REVIEW

**Comment by Soline324** [comment]
* Adjust format of graphic designer

**Comment by Soline324** [comment]
* change location

File: content/about-cn/recruit/product-design/graphic-designer.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+title: 平面视觉设计师
+tags:
+  - Design
```
Comment (YiniXu9506) [inline content/about-cn/recruit/product-design/graphic-designer.md]: This tag should be `Product & Design`

Comment (Soline324) [reply content/about-cn/recruit/product-design/graphic-designer.md]: OK~

**Comment by Soline324** [comment]
* Change tags of graphic-designer

**Pull request merged**